### PR TITLE
Updating some instructions.

### PR DESCRIPTION
**Pull request by ecwillis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,23 +2,39 @@
 
 Well hello there! 👋
 
-We want to assess your Front End proficiency and we've prepared this little test that should not take you a lot of time (ideally). It's a simple Newsletter subscription module.
+We want to assess your Front End proficiency and we've prepared this little test that should not take you a lot of time (ideally). The challenge is to create a simple newsletter aggregator application with a simple subscription form.
```
Comment (dawnerd) [inline README.md]: I think calling it an aggregator app might throw some people off. Should be pretty limited in scope that it's just an opt-in/opt-out form.

File: README.md [hunk 2]
```diff
@@ -2,23 +2,39 @@
...
-We want to see your skills, specifically:
+Our goal is to identify your skill level in the following areas:
 
 * Front End concerns
 * Code semantics
 * Modularity
 * Attention to detail
 
-As a requirement, the module should look good on any device and work properly. There should be at least **one more element to select**. Doesn't have to be pixel perfect and, you're free to use placeholder text instead of what's on the design. 
+As a general requirement, the module should look good on any device and work properly. There should be at least **one more element to select**. Doesn't have to be pixel perfect and, you're free to use placeholder text instead of what's on the design.
```
Comment (dawnerd) [inline README.md]: Let's define what we mean any device. Does this include tvs? What about watches? Lets call out Desktop & mobile.

Also let's be explicit again and say it must contain five or more newsletters.

File: README.md [hunk 3]
```diff
@@ -2,23 +2,39 @@
...
-We want to assess your Front End proficiency and we've prepared this little test that should not take you a lot of time (ideally). It's a simple Newsletter subscription module.
+We want to assess your Front End proficiency and we've prepared this little test that should not take you a lot of time (ideally). The challenge is to create a simple newsletter aggregator application with a simple subscription form.
 
-We want to see your skills, specifically:
+Our goal is to identify your skill level in the following areas:
 
 * Front End concerns
 * Code semantics
 * Modularity
 * Attention to detail
```
Comment (dawnerd) [inline README.md:12]: Also need to make sure we mention performance and accessibility.

File: README.md [hunk 4]
```diff
@@ -2,23 +2,39 @@
...
+To simplify the task, you can simulate that the form takes two seconds to submit and randomly notify that it has been submitted correctly, or that an error has occurred.
 
-Use either plain JavaScript (no TypeScript) or React. The usage of any other library is **completely discouraged** since there should be no need these days. If you're taking the React route, please `useHooks` as that's what we would like to assess. It is fine though (and even encouraged) to use [Create React App](https://github.com/facebook/create-react-app#creating-an-app) if you're taking the React path. 
+## Completion Instructions:
 
-![Composition](/assets/composition.png)
+- Initialize a package using either `yarn` or `npm`.
+- Install any required pacakges and prepare the code and folder structure however you like.
+- _IF_ you are using ES6 or similar code that must be transpiled, add a `build`, `start` or `dev` script to the `package.json`
+- Build the form using your chosen code paradigm. You may use ES5/6, React, or Vanilla Javascript, etc. Any other libraries, or syntax supersets (ie. Typescript) are **completely discouraged**.
```
Comment (dawnerd) [inline README.md]:
```suggestion
- Build the form using your chosen code paradigm. You may use ES5/6, React, or Vanilla Javascript, etc.
```

I'd actually remove that part. If someone opts to use TS or another library they should be ready to explain the choice. We might learn something from it too.

File: README.md [hunk 5]
```diff
@@ -2,23 +2,39 @@
...
+- If you are using React:
+    - Please `useHooks` as that is what we would like to assess.
```
Comment (dawnerd) [inline README.md]: This is confusing as it leads someone to believe this is all we're looking for.

**Review by dawnerd — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (ecwillis) [reply README.md]: How would we word this then to be more generic? This is what Antonio had put, I just formatted it.

**Review by dawnerd — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jgallen23) [reply README.md]: agreed

**Pull request merged**